blog-post.html

```diff
@@ -1,12 +1,98 @@
+    <section id="header">
+      <a href="index.html">
+        <h1>Post Only</h1>
+      </a>
+      <a href="/index.html"></a>
+      <h3>The page for home</h3>
+    </section>
+
+    <section id="main">
+      <h2>There are no mistakes, just happy accidents </h2>
+      <img class="large-image"
+        src="https://www.nationalgeographic.com/content/dam/travel/2019-digital/mount-rainier-national-park/mountain-pine-trees.jpg">
+      <h4>Yeah, that's basically it</h4>
+      <p>No pressure. Just relax and watch it happen. We have no limits to our world. We're only limited by our
+        imagination. If you didn't have baby clouds, you wouldn't have big clouds. Once you learn the technique, ohhh!
+        Turn you loose on the world; you become a tiger. Volunteering your time; it pays you and your whole community
+        fantastic dividends. For the lack of a better word I call them hangy downs. Just relax and let it flow. That
+        easy. Brown is such a nice color.</p>
+      <p>
+        We'll throw some happy little limbs on this tree. Mix your color marbly don't mix it dead. You have freedom
+        here. The only guide is your heart. Let's make a nice big leafy tree. Talk to trees, look at the birds. Whatever
+        it takes. That's crazy. We need dark in order to show light. Let's give him a friend too. Everybody needs a
+        friend.
+      </p>
+      <h4>The least little bit can do so much. Take your time. Speed will come later.</h4>
+      <p>
+        When you buy that first tube of paint it gives you an artist license. It's beautiful - and we haven't even done
+        anything to it yet. We'll do another happy little painting. You want your tree to have some character. Make it
+        special. In this world, everything can be happy. I thought today we would do a happy little picture. Let's build
+        an almighty mountain. If these lines aren't straight, your water's going to run right out of your painting and
+        get your floor wet. How do you make a round circle with a square knife? That's your challenge for the day.
+      </p>
+      </br>
+      </p>
+  </div>
+  </section>
+
+  <section id="footer">
+    <h2> Questions or Comments? Get in Touch:</h2>
+    <input type="name" id="name" placeholder="Name" name="name" />
+    <input type="email" id="email" placeholder="Email" name="email" />
+    <input type="message" id="message" placeholder="Message" name="message" />
+    <button id="sendbutton">Send Message</button>
+    <p class="description">Any little thing can be your friend if you let it be. If it's not what you want - stop and
+      change it. Don't just keep going and expect it will get better. </p>
+    <div class="addresscontainer">
+      <a href="https://www.google.com/maps" target="_blank">
+        <img class="address logo" src="https://icon-library.com/images/home-logo-icon/home-logo-icon-16.jpg"></a>
+      <p class="address"> 1234 Address St. </br>Town, ST 00000, USA </p>

```

Comment:
The given example shows the address taking up 3 lines and keeping the same indentation through the 3 lines rather than having the text word wrap around the image